server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1516,6 +1516,24 @@
         </complexContent>
     </complexType>
 
+    <complexType name="httpProxyServicePropertiesType">
+        <complexContent>
+            <restriction base="gateway:ServicePropertiesType">
+                <sequence>
+                    <element name="use-forwarded" maxOccurs="1" minOccurs="0">
```

Comment:
We couldn't express well enough in schema for other http.proxy service properties. So we are not using something like httpProxyServicePropertiesType. Remove the schema type and  validate in the code.